Resources/Audio/_DV/Lobby/attributions.yml

```
@@ -3,3 +3,13 @@
   license: "CC-BY-SA-4.0"
   copyright: "by Darklyac for DeltaV"
   source: "https://github.com/DeltaV-Station/Delta-v.git"
+
+- files: ["Oliver_Buckland_-_espial.ogg"]
+  license: "CC-BY-SA-3.0"
```

Espial's license is [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/), according to the license link on the bandcamp page. It was probably updated to 4.0 since.

```suggestion
  license: "CC-BY-SA-4.0"
```